demos/chat-server/rust/.cargo/config

```diff
@@ -2,5 +2,5 @@
 target = "wasm32-wasi"
 
 [target.wasm32-wasi]
-runner = ["enarx", "run", "--wasmcfgfile", "Enarx.toml"]
+runner = ["enarx", "run", "--wasmcfgfile", "../Enarx.toml"]
```

Comment:
Can we avoid this diff?
Is it possible for you to extract this into a separate PR?